Doc/library/ssl.rst

```diff
@@ -2748,7 +2748,7 @@ disabled by default.
 
    >>> client_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
    >>> client_context.minimum_version = ssl.TLSVersion.TLSv1_3
-   >>> client_context.maximum_version = ssl.TLSVersion.TLSv1_3
+   >>> client_context.maximum_version = ssl.TLSVersion.MAXIMUM_SUPPORTED
```

Comment:
I think just removing `maximum_version` would automatically make it v1.3+ itself.